test/ellipsis.js

```diff
@@ -1,7 +1,10 @@
 describe('Handlebars.helpers.ellipsis', function() {
 
     var tests = [
-        { text: 'The quick brown fox', len: 12, expected: 'The quick...' }
+        { text: '', len: 1, expected: undefined },
```

Comment:
I'm not really sure if this should be the expected behavior. I'd rather get the input if it couldn't be ellipsed (but that's off topic here, maybe)